modules/au.com.trgtd.tr.task.messages/test/unit/src/au/com/trgtd/tr/task/messages/MessageParserTest.java

```diff
@@ -0,0 +1,126 @@
+package au.com.trgtd.tr.task.messages;
+
+import au.com.trgtd.tr.task.messages.MessageParser.Message;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.List;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MessageParserTest {
+
+    private MessageParser parser = new MessageParser();
+    private LocalDate localDate = LocalDate.of(2014, Month.MARCH, 26);
```

Comment:
Why this date? What about introduce some variable with proper name explaining why this date?